### ISSUE TITLE
BottomBar: customizable inactive color

### ISSUE THREAD

**Issue by jozan** [callstack/react-native-paper#366]
I really like BottomBar but it's not possible to customise inactive color at all. Are there any plans to support it or are there any objections if I send a PR?

I'm using BottomBar via [react-navigation-material-bottom-tab-navigator](https://github.com/react-navigation/react-navigation-material-bottom-tab-navigator).

References:

![image](https://user-images.githubusercontent.com/388265/39955501-6c867da4-55d8-11e8-81bd-fbee39b270bd.png)

```js
const inactiveColor = shifting
  ? textColor
  : color(textColor)
      .alpha(0.5)
      .rgb()
      .string();
```

https://github.com/callstack/react-native-paper/blob/master/src/components/BottomNavigation.js#L521

**Comment by satya164** [comment]
I'd merge a PR adding `inactiveTintColor` and `activeTintColor` props. Wanna send a PR?

**Comment by jozan** [comment]
Yes, I’ll start working on it in a few days once I’ve finished some other
stuff.

**Comment by jozan** [comment]
@satya164 Seem like you've already implemented this on 2.0. (#371)


https://github.com/callstack/react-native-paper/pull/371/files#diff-18fcf70e87723a21038fbb839fddd4a2R153
